docs/app/pages/charts/sections/spark-charts/spark-charts/snippets/app.html

```diff
@@ -5,33 +5,33 @@
 
     <div class="col-md-1 col-sm-1 col-xs-1"></div>
 
-    <ux-spark [barColor]="charts[1].barColor" [trackColor]="charts[1].trackColor" [value]="charts[1].value" [barHeight]="charts[1].barHeight"
+    <ux-spark theme="chart3" [value]="charts[1].value" [barHeight]="charts[1].barHeight"
```

Comment:
As you have now defined the theme as part of the object in the class you should reference it instead, eg:

```html
<ux-spark [theme]="charts[1].theme"
```